src/NuGet.Clients/NuGet.PackageManagement.UI/Options/AddMappingDialog.xaml

```diff
@@ -0,0 +1,125 @@
+<nuget:VsDialogWindow x:Class="NuGet.Options.AddMappingDialog"
```

Comment:
Please add some more space on buttons to reduce text overflow possibility in non-English strings.

![Text overflow in Package Source Mapping Options page](https://user-images.githubusercontent.com/1192347/183227605-f7ddc003-ff6a-4ebb-8064-1da3ce3f267a.png)

Reply:
https://github.com/NuGet/Client.Engineering/issues/1834